samza-kafka/src/main/scala/org/apache/samza/config/KafkaConfig.scala

```diff
@@ -88,6 +88,8 @@ object KafkaConfig {
 
   val DEFAULT_RETENTION_MS_FOR_BATCH = TimeUnit.DAYS.toMillis(1)
 
+  val MIN_COMPACTION_LOG_MS = "min.compaction.lag.ms"
```

Comment:
Typo in variable name: LAG_MS instead of LOG_MS

Reply:
Fixed.